src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,7 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+
```

Comment:
you need to read file with typical mistakes. You need to add some fields and initialize those using super(...);